region/wreckedship/main.json

```diff
@@ -1490,6 +1490,25 @@
               "openEnd": 1
             }
           ],
+          "canLeaveCharged": [
+            {
+              "framesRemaining": 70,
+              "usedTiles": 33,
+              "openEnd": 2,
+              "strats": [
+                {
+                  "name": "XMode",
+                  "notable": false,
+                  "requires": [
+                    "h_canXMode",
+                    "canIframeSpikeJump",
+                    "canShinechargeMovement",
+                    {"spikeHits": 3}
```

Comment:
I believe this will only work if phantoon is defeated? (the strats to 6 shouldnt need it because 6->3 requires it)

Reply:
It must have been later that  I confirmed phantoon must be defeated for these spikes to be active.
I was testing SuperJumps that we can look at adding after this PR.